my_internet_speed/settings.py

```diff
@@ -18,7 +18,7 @@
 )
 
 CELERY_CONFIG = {'broker': REDIS_URL, 'timezone': TIMEZONE}
-
+POST_TWITTER=config('POST_TWITTER', default=False, cast=bool)
```

Comment:
Instead of having this configuration, what about:

```python
CREDENTIALS = {
    'consumer_key': config('TWITTER_CONSUMER_KEY', default=None),
    'consumer_secret': config('TWITTER_CONSUMER_SECRET', default=None),
    'access_token': config('TWITTER_ACCESS_TOKEN', default=None),
    'access_token_secret': config('TWITTER_ACCESS_TOKEN_SECRET', default=None)
}
TWITTER = Api(**CREDENTIALS) if all(CREDENTIALS.values()) else None
```